src/PP.Domain/IDataSeedContributor.cs

```diff
@@ -10,64 +11,71 @@ namespace PP
             _attractionRepository = attractionRepository;
             _rideRestrictionRepository = ridestrictionRepository;
-        }
-
 
+        }
 
         public async Task SeedAsync(DataSeedContext context)
         {
-            if(await _attractionRepository.GetCountAsync() <= 0)
+            if (await _attractionRepository.GetCountAsync() > 0)
             {
-                await _attractionRepository.InsertAsync(new Attraction
-                {
-                    Name ="Bob Express",
-                    Type = AttractionType.Rollercoaster,
-                },
-                autoSave: true
-                );
+                return;
+            }
 
-                await _attractionRepository.InsertAsync(new Attraction
-                {
-                    Name="El Rio",
-                    Type = AttractionType.Waterattractie,
-                },
-                autoSave: true
-                );
-                
+            var bobExpress = await _attractionRepository.InsertAsync(new Attraction
+            {
+                Name = "Bob Express",
+                Type = AttractionType.Rollercoaster,
             }
+             );
 
-            if (await _rideRestrictionRepository.GetCountAsync() <= 0)
+            var elRio = await _attractionRepository.InsertAsync(new Attraction
             {
-                await _rideRestrictionRepository.InsertAsync(new RideRestriction
-                {
-                    Name = "1.2m Volwassen",
-                    Type = RideRestrictionType.Adult,
-                    ValueLow = 1200,
-                    ValueHigh = 3000
-                },
-                autoSave: true
-                );
+                Name = ELRIO,
+                Type = AttractionType.Waterattractie,
+            }
+
+             );
 
-                await _rideRestrictionRepository.InsertAsync(new RideRestriction
-                {
-                    Name = "90 cm - 1m20 begeleiding",
-                    Type = RideRestrictionType.Supervision,
-                    ValueLow = 900,
-                    ValueHigh = 1200
-                },
-                autoSave: true
-                );
+            if (await _rideRestrictionRepository.GetCountAsync() > 0)
+            {
+                return;
             }
+
+            var Volwassen12m = await _rideRestrictionRepository.InsertAsync(new RideRestriction
+            {
+                Name = "1.2m Volwassen",
+                Type = RideRestrictionType.Adult,
+                ValueLow = 1200,
+                ValueHigh = 3000
+            }
+            );
+
+            var begeleiding = await _rideRestrictionRepository.InsertAsync(new RideRestriction
+            {
+                Name = "90 cm - 1m20 begeleiding",
+                Type = RideRestrictionType.Supervision,
+                ValueLow = 900,
+                ValueHigh = 1200
+            }
+            );
+
+            elRio.RideRestrictions.Add(Volwassen12m);
```

Comment:
SImple as it should be

Reply:
Ik wist dat het iets simpel ging zijn. Maar hier had ik niet aan gedacht.